plugins/dependencies/src/main/resources/refreshVersions-rules/testing-version-alias-rules.txt

```diff
@@ -7,9 +7,6 @@ org.junit.jupiter:junit-jupiter(-*)
 io.kotest:*
    ^^^^^^
 
-io.kotest.extensions:kotest-extensions-*
-                     ^^^^^^^^^^^^^^^^^
-
```

Comment:
This should not be removed, but edited instead so it resolves to `version.kotest`.